cloud/services/scalesets/scalesets_test.go

```diff
@@ -377,6 +377,80 @@ func TestReconcileVMSS(t *testing.T) {
 				}))
 			},
 		},
+		{
+			name:          "can create a vmss with a Windows node",
```

Comment:
heads up @devigned this conflicts with #1067

Reply:
Is there anything I need to do here?